client/src/main/com/sinch/sdk/domains/verification/adapters/StatusService.java

```diff
@@ -30,16 +28,10 @@ protected QueryVerificationsApi getApi() {
     return this.api;
   }
 
-  public VerificationReport get(Identity identity, VerificationMethodType method) {
-    if (!(identity instanceof NumberIdentity)) {
-      throw new ApiException("Unexpected entity: " + identity);
-    }
-    NumberIdentity numberIdentity = (NumberIdentity) identity;
+  public VerificationReport get(NumberIdentity identity, VerificationMethodType method) {
 
     return VerificationsDtoConverter.convert(
-        getApi()
-            .verificationStatusByIdentity(
-                numberIdentity.getType(), numberIdentity.getEndpoint(), method.value()));
+        getApi().verificationStatusByIdentity("number", identity.getEndpoint(), method.value()));
```

Comment:
Yes by design it was removed by current PR.
The `numberIdentity.getEndpoint()` do not have to be public; this is an implementation detail because of part of the URL to be addressed by service.
From SDK's user perspective it does not have sense to having access to it and could cause misunderstanding or worst could be used because of public now and will cause issue with SDK usage if change in the future.
Even the `NumberIdentity.TYPE` will have to be public and could cause same kind of issue later.
The only way to avoid this being public but not hardcoded will come when design started with Conversation will spread onto this service: `models` as `interfaces` and implementation onto adapters side with known-only-by-service-implementation contracts